### ISSUE TITLE
For Gowalla dataset

### ISSUE THREAD

**Issue by shmizhang** [allenjack/SAE-NAD#1]
Hello. On the Gowalla dataset, we set the network architecture as [N, 500, 50, 500, N]. However,  we get a  strange result: the precision@k is higher than yours, and the recall@k is lower than yours. I tried to adjust the ratio of the test set and the training set, but still can't reach the results of your experiment. Have you encountered such an issue? How do you solve this issue? I would be very grateful if you could give me an early reply.

**Comment by allenjack** [comment]
Hi,
Thank you for your interests. 
I am wondering are you using the Gowalla dataset that preprocessed by yourselves? If that is the case, the results can be different. Could you write down your email then I can send you the Gowalla dataset that we used in the paper?

**Comment by shmizhang** [comment]
Thanks a lot. My email is  shmizhang@gmail.com.
